From cd89c9d2f0a1692a8e12b7c00ba93168de716e0a Mon Sep 17 00:00:00 2001
Patch-Source: https://github.com/dotnet/source-build/issues/2695
From: Antoine Martin <dev@ayakael.net>
Date: Thu, 20 Jan 2022 14:24:10 +0000
Subject: [PATCH 1/1] roslyn_added-previously-built-artifacts-repo

On rebuild, roslyn cannot seem to find certain nuget packages that are
usually provided by artifacts. Thus, this adds a nuget repo that points
to a yet-to-be-defined /path/to/artifacts. APKBUILD, using sed, adds
correct path to /usr/share/dotnet/artifacts/$_artifactsver in
default_prepare() function
---
 NuGet.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/NuGet.config b/NuGet.config
index 3cd2f6f4e6a..80f080d84e6 100644
--- a/NuGet.config
+++ b/NuGet.config
@@ -3,6 +3,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="previously-source-built" value="/path/to/artifacts/" />
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
-- 
2.34.1

